libs/td/Makefile

```diff
@@ -0,0 +1,75 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=td
+PKG_VERSION:=1.8.0
+PKG_RELEASE:=$(AUTORELEASE)
+PKG_LICENSE:=Boost Software License - Version 1.0
+PKG_LICENSE_FILES:=LICENSE_1_0.txt
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_GIT_URL:=github.com/tdlib/td
+PKG_GIT_REF:=v$(PKG_VERSION)
+PKG_SOURCE_URL:=https://codeload.$(PKG_GIT_URL)/tar.gz/$(PKG_GIT_REF)?
+PKG_HASH:=30d560205fe82fb811cd57a8fcbc7ac853a5b6195e9cb9e6ff142f5e2d8be217
+
+PKG_BUILD_DEPENDS:=gperf/host $(PKG_NAME)/host
+PKG_BUILD_PARALLEL:=1
+CMAKE_BINARY_SUBDIR:=openwrt-build
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+EXTRA_CFLAGS += -Wno-psabi
+
+define Package/td
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=td
```

Comment:
Please use here the full title:
`Telegram Database library`

Reply:
Better now or still something missing?